### PR TITLE
making attachments injectable

### PR DESCRIPTION
**Pull request by Lonwyr**
To enable the abstract-planet-attachment in homebrew, the dataset currently directly derived from the attachment.data has to be extendable.
In addition, an internal mapping war introduced for easier consumption (getByCardNsidName / getByTokenNsidName).

### CODE REVIEW

**Comment by Lonwyr** [comment]
NOT YET COMPLETLY TESTED - this is a "call for feedback" and this will be tested in ttpg in the upcoming hours & days

File: src/objects/attachments/attachment.schema.js
Comment (Lonwyr) [inline src/objects/attachments/attachment.schema.js:1]: for the images it may be required to add a packaceId...

**Comment by darrellanderson** [comment]
Looks like a conflict with global.js needs to be merged

**Comment by Lonwyr** [comment]
packageId determination was broken... now all works like a charm

image

**Pull request merged**